apps/google-analytics-4/frontend/src/components/config-screen/map-account-property/MapAccountPropertyDropdown.tsx

```diff
@@ -55,51 +56,47 @@ const MapAccountPropertyDropdown = (props: Props) => {
-                </Select.Option>
-                {sortedAccountSummaries.map((accountSummary: AccountSummariesType) => (
-                  <optgroup
-                    label={`${accountSummary.displayName} (${getIdOnly(accountSummary.account)})`}
-                    key={accountSummary.account}>
-                    {accountSummary.propertySummaries.map((propertySummary) => (
-                      <Select.Option
-                        key={propertySummary.property}
-                        value={propertySummary.property}>
-                        {`${propertySummary.displayName} (${getIdOnly(propertySummary.property)})`}
-                      </Select.Option>
-                    ))}
-                  </optgroup>
-                ))}
-              </Select>
-              <FormControl.HelpText marginTop="spacingM">
-                If you don't see a property in the dropdown, make sure the Google service account
-                installed above has been given "viewer" access to it in Google Analytics.{' '}
-                <em>Note:</em> Only Google Analytics 4 properties are listed; Google "Universal
-                Analytics" properties are not supported. See{' '}
-                <TextLink
-                  href="https://support.google.com/analytics/answer/10759417"
-                  target="_blank"
-                  icon={<ExternalLinkIcon />}
-                  alignIcon="end">
-                  "Make the switch to Google Analytics 4"
-                </TextLink>{' '}
-                for more information.
-              </FormControl.HelpText>
-            </FormControl>
-          </Box>
+        <Stack spacing="spacingXs" flexDirection="column" alignItems="flex-start">
+          <Stack spacing="spacingXs">
+            <WarningDisplay warningType={warningType} tooltipContent={tooltipContent} />
+            <Select
+              testId="accountPropertyDropdown"
+              value={selectionValue}
+              onChange={onSelectionChange}
+              isInvalid={warningType === WarningTypes.Error}>
+              <Select.Option key="empty option" value="" isDisabled>
+                Select a property...
+              </Select.Option>
+              {sortedAccountSummaries.map((accountSummary: AccountSummariesType) => (
+                <optgroup
+                  label={`${accountSummary.displayName} (${getIdOnly(accountSummary.account)})`}
+                  key={accountSummary.account}>
+                  {accountSummary.propertySummaries.map((propertySummary) => (
+                    <Select.Option key={propertySummary.property} value={propertySummary.property}>
+                      {`${propertySummary.displayName} (${getIdOnly(propertySummary.property)})`}
+                    </Select.Option>
+                  ))}
+                </optgroup>
+              ))}
+            </Select>
+          </Stack>
+          <Paragraph marginTop="spacingM" style={{ color: tokens.gray600 }}>
```

Comment:
It's super annoying because it's such a tiny thing but I think we should move this to a `MapAccountPropertyDropdown.styles.tsx` file like we do elsewhere (e.g. https://github.com/contentful/apps/blob/master/apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentType.styles.ts), just so we avoid adhoc css creeping in.

Reply:
No worries, makes sense! Updating now.